umap/static/umap/js/modules/umap.js

```diff
@@ -1181,7 +1181,7 @@ export default class Umap {
     }
     this._advancedActions(container)
 
-    this.editPanel.open({
+    return this.editPanel.open({
```

Comment:
Is it related to that PR?

Reply:
Yes, it return the promise so someone calling open() can use this promise to run some code when the panel is opened (in this case we'll open a given fieldset) :)